odoo/tools/sql.py

```diff
@@ -313,25 +313,32 @@ def reverse_order(order):
     return ', '.join(items)
 
 
-def increment_field_skiplock(record, field):
+def increment_fields_skiplock(record, *record_fields):
     """
-        Increment 'friendly' the [field] of the current [record](s)
+        Increment 'friendly' the given field(s) of the current record(s)
         If record is locked, we just skip the update.
         It doesn't invalidate the cache since the update is not critical.
 
-        :rtype: bool - if field has been incremented or not
+        :param record: recordset to update
+        :param record_fields: integer fields to increment
+        :rtype: bool - if field(s) has (have) been incremented or not
     """
     if not record:
         return False
 
-    assert record._fields[field].type == 'integer'
+    for field in record_fields:
+        assert record._fields[field].type == 'integer'
+
+    fields_update = ', '.join(f'{field} = {field} + 1' for field in record_fields)
 
     cr = record._cr
     query = """
-        UPDATE {table} SET {field} = {field} + 1 WHERE id IN (
-            SELECT id from {table} WHERE id in %(ids)s FOR UPDATE SKIP LOCKED
-        ) RETURNING id
-    """.format(table=record._table, field=field)
+        UPDATE {table}
+           SET {fields_update}
+         WHERE id 
+            IN (SELECT id FROM {table} WHERE id in %(ids)s FOR UPDATE SKIP LOCKED)
+     RETURNING id
```

Comment:
Remove returning, we're not actually using it (the result could just be a `bool(cr.rowcount)`) and we have no signal for partial updates.